ground/src/main/proto/com/google/ground/submission.proto

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option java_package = "com.google.android.ground.proto";
+option java_multiple_files = true;
+
+import "google/protobuf/timestamp.proto";
+import "com/google/ground/audit_info.proto";
+import "com/google/ground/geometry.proto";
+
+// Data submitted by data collectors for relevant tasks in a job.
+message Submission {
+    // Required. A system-defined ID.
+    string id = 1;
+
+    // Required. The ID of the location of interest to which this submission
+    // refers.
+    string loi_id = 2;
+
+    // Required. The ID of the job to which this submission refers. This can
+    // also be obtained from the location of interest, but is duplicated here
+    // to facilitate filtering.
+    string job_id = 3;
+
+    // Required. Info about who created this submission and when.
+    AuditInfo created = 4;
+
+    // Info about who last modified this submission and when. If unset,
+    // the submission was never modified after it was created.
+    AuditInfo last_modified = 5;
+
+    // The data collector's responses to tasks in this job.
+    repeated TaskData task_data = 6;
+}
+
+// Data provided by data collectors for a specific task and LOI.
+message TaskData {
+    // Required. A system-defined ID.
+    string id = 1;
+
+    // Required. ID of the task to which this task data refers.
+    string task_id = 2;
+
+    // Indicates this task was skipped pr
```

Comment:
Typo?

Reply:
Oops.. the `skipped` field got deleted for some reason, adding it back!